src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css/index.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA&sol;A)"
+      >
+
+      <h3 class="card__title">
+        APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA&sol;A)
+      </h3>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__stars">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
+
+          <li class="stars__star"></li>
```

Comment:
No needs to add empty line between _oneline_ siblings
```suggestion
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
          <li class="stars__star"></li>
```